csharp/compatibility_tests/v3.0.0/test.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+function run_test() {
+  # Generate test proto files.
+  ./protoc_1 -Iprotos/src -I../../../src/ --csharp_out=src/Google.Protobuf.Test \
+    --csharp_opt=base_namespace=Google.Protobuf \
+    protos/src/google/protobuf/map_unittest_proto3.proto \
+    protos/src/google/protobuf/unittest_import_proto3.proto \
+    protos/src/google/protobuf/unittest_import_public_proto3.proto \
+    protos/src/google/protobuf/unittest_well_known_types.proto
+
+  ./protoc_1 -Iprotos/csharp --csharp_out=src/Google.Protobuf.Test \
+    --csharp_opt=base_namespace=UnitTest.Issues \
+    protos/csharp/protos/unittest_issues.proto
+
+  ./protoc_2 -Iprotos/src --csharp_out=src/Google.Protobuf.Test \
+    --csharp_opt=base_namespace=Google.Protobuf \
+    protos/src/google/protobuf/unittest_proto3.proto
+
+  # Build and test.
```

Comment:
Not an issue for now, but something we'll need to work around at some point: the project.json format used for the current code is going away, to be replaced by csproj. At some point we'll want to update to that, and we'll need to do it for this old code too.